src/io/flutter/FlutterBundle.properties

```diff
@@ -219,7 +219,7 @@ settings.format.code.on.save=Format code on save
 settings.organize.imports.on.save=Organize imports on save
 settings.flutter.version=Version:
 settings.open.inspector.on.launch=Open Flutter Inspector view on app launch
-settings.hot.reload.on.save=Perform hot reload on save
+settings.hot.reload.on.save=Perform hot reload on Save All
```

Comment:
Can we instead keep this as `save` (saving generally, and not the specific IntelliJ action), and call out the `Save All` action in tooltip text?

Reply:
SGTM.